docs/admin.rst

```diff
@@ -403,6 +405,34 @@ occurrence of an update to the server, it may not reveal the content of the chan
 management option would be SSHing into the server and manually inspecting the configuration to
 identify any discrepancies.
 
+Configuring Localization for the *Source Interface* and the *Journalist Interface*
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The *Source Interface* and *Journalist Interface* are translated in the following
+languages:
+
+.. include:: includes/l10n.txt
+
+At any time during and after initial setup, you can chose from a list of
+supported languages to display using the codes shown in parentheses.
+
+.. note:: With a *Source Interface* displayed in French (for example), sources
+          submitting documents are likely to expect a journalist fluent in
+          French to be available to read the documents and follow up in that
+          language.
+
+To add or remove locales from your instance, you'll need to :ref:`update your
+system configuration <update-system-configuration>` as outlined above.
+
+When you reach the prompt starting with "Space separated list of additional
+locales to support", you will see a list of languages currently supported.
```

Comment:
You might want to explicitly tell folks that they need to update the list at this point, adding or removing entries as appropriate.